campaign/4e/char_power.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+  Please see the license.html file included with this distribution for 
+  attribution and copyright information.
+-->
+
+<root>
+    <windowclass name="char_power" merge="join" ruleset="4E">
+		<script>
+            local fActivatePowerOriginal;
+            function onInit()
```

Comment:
I'm honestly a bit torn on this. On the one hand I'd like the larger script block to be in a lua file so the IDE can behave a bit better with it. On the other that would then make this file like 5 lines long, which seems perhaps a bit silly.

Reply:
I moved them into their own scripts. It's cleaner at least.